src/app/pages/home/home.component.html

```diff
@@ -1,7 +1,7 @@
 <main>
 	<section class="card-decks mt-16 mb-16">
 		@if (!!previews && previews.length) {
-			<ng-container *ngFor="let storylistDeck of previews">
```

Comment:
Está bien el cambio en lo que respecta a pasar de *ngFor a @for, pero el `track` no debería ser nunca un objeto como es en este caso.

Recomiendo transformar la expresión incluida en `track` a `$index` o, en el mejor de los casos, a `storylistDeck.slug`, dado que los slugs son propiedades que garantizadamente poseen unicidad para todos los distintos elementos incluidos en el gestor de contenidos.

Reply:
Gracias por la aclaración,modificado a storylistDeck.slug.